.env.example

```diff
@@ -21,6 +21,7 @@ DEPLOYER_PRIVATE_KEY=
 #ARCHIVE_NODE_opsepolia=https://sepolia.optimism.io
 #ARCHIVE_NODE_opmainnet=https://opt-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>
 #ARCHIVE_NODE_basesepolia=https://sepolia.base.org
+#ARCHIVE_NODE_basemainnet=https://mainnet.base.org
```

Comment:
```suggestion
#ARCHIVE_NODE_basemainnet=https://open-platform.nodereal.io/<YOUR_KEY_HERE>/base
```

Reply:
I would suggest using the node real endpoint for mainnet

Reply:
https://github.com/VenusProtocol/venus-protocol/pull/535/commits/405b4dfeabfa8b02b2a970ef5fad535646e480c9